Update client config

CO2-code opened the pull request:
Updated the resources ...👍

CO2-code committed:
Resources Replaced ...

Metadorius commented on Resources/ClientDefinitions.ini:
```diff
@@ -24,11 +27,11 @@ GameLauncherExecutableName=ra95.dat
+SettingsFile=REDALERT.ini
 ; Hotkey configuration file for the hotkey configuration window in the options menu.
-KeyboardINI=REDALERT.INI
+KeyboardINI=KeyboardMD.ini
```
This isn't correct, `KeyboardMD.ini` is the file for Yuri's Revenge

CO2-code replied:
What should i do for it ?

CnCRAZER replied:
Find the ini file associated with RA1 keybinds and then declare it

Metadorius commented on Resources/language_800x600.dll:
Not sure what this dll is for?

CO2-code replied:
I created it because whatever the In-game Resolution I choose from Options , it says : 
Saving Settings Failed , Could not find language_800x600.dll.
Same thing with other resolutions but it changes according to the res ( like language_1280x720.dll)

Metadorius commented on Resources/ts_ddraw.dll:
same, wasn't there already TS-DDraw included?

CO2-code replied:
Added it by mistake 😚

Metadorius commented on Resources/GameCollectionConfig.ini:
You can delete this file, since we have the RA definition hardcoded inside the client -- we don't need this.

Metadorius reviewed (changes requested):
Some changes in ClientDefinitions look a bit arbitrary, but w/e.

CO2-code committed:
removed unnecessary files

CnCRAZER commented on package/Resources/ClientDefinitions.ini:
```diff
@@ -89,20 +92,20 @@ CreateSavedGamesDirectory=yes
-ModMode=false
-
-CopyResolutionDependentLanguageDLL=false
+ModMode=true
```
Should this be set to True?

CO2-code committed:
Update :)

CO2-code committed:
Updates....

CO2-code committed:
moved config. files from theme Default to Resources folder.

CO2-code committed:
DTACnCNetClient.ini must be in ThemeDefault

CO2-code committed:
updates.

CO2-code committed:
replace Resources/ThemeDefault/LANGameLobby.ini

CO2-code committed:
Add files via upload

CO2-code committed:
Map preview/ binaries updated.

CO2-code committed:
Map bug fixed.

CO2-code committed:
Merge branch 'master' of https://github.com/CO2-code/cncnet-ra-client-package

CO2-code committed:
Create LANGameLobby.ini

CO2-code committed:
Delete LANGameLobby.ini

mistake

MahBoiDeveloper committed:
Move theme non-related ini's to the `Resources` folder

MahBoiDeveloper committed:
Fix `Got It` button position and enable border draw

MahBoiDeveloper committed:
Enable borders drawing

MahBoiDeveloper committed:
Update client binaries to 2.12.12 version

CO2-code committed:
ClientGameType changed to RA, added voiceicon.png and deleted client crash  logs .

MahBoiDeveloper committed:
Update client binaries to 2.12.13 version

MahBoiDeveloper committed:
Update regular and bold sprite fonts with latet CJK changes

CO2-code committed:
maps

CO2-code committed:
Add files via upload

Removed Credits button ...

CO2-code committed:
Add files via upload

CO2-code committed:
Maps++

CO2-code committed:
Add files via upload

MahBoiDeveloper committed:
Update client stub to the latest version (2.0.10.0)

CO2-code committed:
Add files via upload

CO2-code committed:
Add files via upload

MahBoiDeveloper committed:
Enable start position dropdowns and refactoring `GameLobbyBase.ini`

CO2-code committed:
Add files via upload

GameLobbyBase.ini , suitable version for Skirmish and Online game rooms

CO2-code committed:
Game lobby designed/modified by Razer

Co-authored-by: RAZER <79311432+CnCRazer@users.noreply.github.com>

CO2-code committed:
Game lobby designed/modified by Razer

CO2-code committed:
Merge branch 'master' of https://github.com/CO2-code/cncnet-ra-client-package

CnCRAZER committed:
Initial Port

CnCRAZER committed:
Add game search filter for online lobbies

CnCRAZER committed:
Lobby UI Adjustments + Housekeeping

CnCRAZER committed:
Add msgboxform.png

CnCRAZER committed:
Update ClientDefinitions.ini

CnCRAZER committed:
Update MultiplayerGameLobby.ini

CnCRAZER committed:
Fix Borders

CnCRAZER committed:
Restructure to mimic YR repo

CnCRAZER committed:
maps

CnCRAZER committed:
Update SkirmishLobby.ini

CO2-code committed:
Binaries updated , General UI fixes

CnCRAZER committed:
Fix Background Texture

DrawMode to Stretch for background texture

CnCRAZER committed:
Refactor to mimic YR repo structure

CnCRAZER committed:
Beginning stages of mission file structure

CO2-code committed:
not sure , some maps..

CnCRAZER committed:
Start of tools folder support

CO2-code committed:
maxname=11

CnCRAZER committed:
Update ClientDefinitions.ini

CnCRAZER committed:
Update .gitignore

CnCRAZER committed:
Merge branch 'Local-Tests' into master

CnCRAZER committed:
Revert "Merge branch 'Local-Tests' into master"

This reverts commit eddfb9730bebfab84eed0404a9f1f2858a14d8bb, reversing
changes made to cfc5be9f2f0828af6fb9b82e490bede58d1f8c74.

CO2-code committed:
Update GameLobbyBase.ini

add colors icons :)

CO2-code committed:
Rename sovieticon.png to random sovietsicon.png

CO2-code committed:
Rename alliedicon.png to random alliesicon.png

CO2-code committed:
Update GameOptions.ini

white , black and dark blue colors

CO2-code committed:
Update ClientDefinitions.ini

MaxNameLength=15 ;11

CO2-code committed:
Update GameOptions.ini

New Colors :)

CO2-code committed:
Delete package/Resources/Binaries directory

CO2-code committed:
Delete package/Resources/BinariesNET8 directory

CO2-code committed:
Update to 2.12.16

CO2-code committed:
FF's maps

 those maps came with the installation , but we have a lot of custom map packages.

CO2-code committed:
Update CnCNetLobby.ini

CO2-code committed:
Update LANLobby.ini

CO2-code committed:
latest Resources

MahBoiDeveloper commented:
Changes has been merged to the `feature/port-to-xna` branch.
